docs/api-references/categories.md

```diff
@@ -0,0 +1,184 @@
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        },
+        {
+            "id": 9,
+            "name": "Hermann Ltd",
+            "description": "Vel consequatur quia quae. Aut quo saepe non. Sint distinctio ipsa voluptas earum exercitationem nemo voluptas eos.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        },
+        {
+            "id": 10,
+            "name": "Lubowitz PLC",
+            "description": "Porro tempore et molestiae ex modi praesentium non. Quia tempora magnam est eveniet dolorum.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        },
+        {
+            "id": 11,
+            "name": "Gorczany-Brekke",
+            "description": "Quidem aspernatur accusamus nihil rerum ea consequatur molestiae molestiae. Porro nostrum ab velit nihil. Voluptatem quasi laudantium libero rem voluptatibus. Ea quidem quidem dicta eius molestiae sit. Quia aut quia quo fugiat voluptas et voluptatum.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        },
+        {
+            "id": 12,
+            "name": "Rice PLC",
+            "description": "Maxime maxime molestiae fugit culpa. Minima vel dolor sed voluptatum aut dignissimos est. Qui eos expedita quia accusantium.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        },
+        {
+            "id": 13,
+            "name": "Marks and Sons",
+            "description": "Placeat itaque voluptatem animi perferendis quis tempora est. Praesentium nihil voluptatem et tempore. Cupiditate nisi consequatur est praesentium libero. Laboriosam quas eius et atque dolores qui.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        },
+        {
+            "id": 14,
+            "name": "Gorczany and Sons",
+            "description": "Quidem est sit dolore sed deleniti voluptatem. Hic ex placeat fugiat aut blanditiis rerum. Architecto vel reprehenderit est molestiae repellendus corporis veniam. Fugit amet ut inventore.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        },
+        {
+            "id": 15,
+            "name": "Abbott, Torp and Heller",
+            "description": "Eveniet corporis dolorum iste sit et enim ea. Unde est ducimus aut minus repudiandae. Ex architecto cum aut omnis est earum sunt. Voluptatibus quisquam ullam reprehenderit aliquam.",
+            "created_at": "2017-07-31 13:58:02",
+            "updated_at": "2017-07-31 13:58:02"
+        }
+    ],
+    "success": true
+}
+```
+
+### `GET` Food by Category
+```
+/categories/{id}
```

Comment:
I think this is not a good endpoint.
`/categories/{id}` by Restful is show detail of category has ID `{id}`, don't display foods of category `{id}`
You can try to use: **GET** `/categories/{id}/foods`